JavaAssignments/src/saket/SquareandCube.java

```diff
@@ -0,0 +1,28 @@
+package saket;
+
+public class SquareandCube {
+
+	int num = 2;
+	int square;
+	int cube;
+
+	void square() {
+
+		square = num * num;
+
+	}
+
+	void cube() {
+		cube = square * num;
+
+	}
+
+	public static void main(String[] args) {
+		SquareandCube A = new SquareandCube();
+		A.square();
```

Comment:
You should take parameters from user and pass into square and cube methods. This method should return value.

int square(int length)
{
        return length*length;
}

Reply:
done the changes